RRS/customer/urls.py

```diff
@@ -10,8 +10,8 @@
         name  = 'index'
     ),
     url(
-        regex = r'^(\d+)/$',
+        regex = r'restaurant/(\d+)$',
         view = views.restaurant,
-        name="restaurant"
+        name="view-restaurant"
```

Comment:
I also had to read on the recommended ways to use spaces on URLs. Turns out that using "-" is the recommended way. But using a single word is still the easiest XD

https://support.google.com/webmasters/answer/76329?hl=en
http://searchengineland.com/google-bing-handle-underscores-dashes-differently-89672